src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2640,4 +2641,8 @@ Delete the following packages: {2}?
   <data name="PackageFilter_Name_All" xml:space="preserve">
     <value>All</value>
   </data>
+  <data name="PreferencesViewShowPreviewBubbles" xml:space="preserve">
```

Comment:
by adding this can you remove `DynamoViewSettingsMenuShowPreviewBubbles` which is pointing to same string value?